### PR TITLE
feature/FOUR-16640

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
Self Service stops filtering after a search on Welcome screen 

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-16640

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next
.

### CODE REVIEW

File: src/components/renderer/form-tasks.vue [hunk 1]
```diff
@@ -208,7 +208,7 @@ export default {
         }
 
         if (this.pmqlSearch) {
-          pmql = pmql + "AND" + this.pmqlSearch;
+          pmql =  pmql ? pmql + "AND" : "" + this.pmqlSearch;
```
Comment (fagubla) [inline src/components/renderer/form-tasks.vue:211]: remove the extra before **pmql**

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-0e1f940176.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-0e1f940176.engk8s.processmaker.net

File: src/components/renderer/form-tasks.vue [hunk 2]
```diff
@@ -208,7 +208,8 @@ export default {
         }
 
         if (this.pmqlSearch) {
-          pmql = pmql + "AND" + this.pmqlSearch;
+        console.log("still here");
```
Comment (fagubla) [inline src/components/renderer/form-tasks.vue:211]: remove console

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-0e1f940176.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-0e1f940176.engk8s.processmaker.net

**Pull request closed without merging**